spec/renuo/bin-check/servant_thread_spec.rb

```diff
@@ -13,10 +15,21 @@
   context 'running successfully' do
     let(:script) { build :script, script_command: './spec/spec-files/test_script_exit0' }
     let(:servant) { RenuoBinCheck::ServantThread.new(script) }
-    let(:result_attributes) { attributes_for :result }
 
     it 'starts the command defined in ScriptConfig and returns a Result' do
       expect(servant.run).to have_attributes(result_attributes)
     end
   end
+
+  context 'finding cache' do
+    let(:script) do
+      build :script, script_command: 'script_name',
```

Comment:
don't you want to use a factory for this? otherwise you can also use the constructor directly?